boot_xbe/xbe.S

```diff
@@ -47,7 +47,7 @@ certificate:
 
   // title name (unicode string, 40 chars) *unimportant, but beautiful*
 unicodename:
-  .word 'L','i','n','u','x',0,0,0,0,0,0
+  .word 'X','r','o','m','w','e','l','l',0,0,0
```

Comment:
Would you mind changing this to **C**romwell instead of Xromwell? After previous discussion on Discord (I think) we agreed that this just adds confusion. Either Cromwell is loaded from an XBE or it's loaded on boot, either way it is still Cromwell.

Reply:
There are other instances of Xromwell in the repository (README at least), so I'd suggest to do this rebranding in a separate PR (before or after this one).